ansible/roles/openshift_udp_limits/tasks/main.yml

```diff
@@ -12,18 +12,8 @@
   delegate_to: "{{ cluster_master }}"
   run_once: true
 
-- name: "Get the cluster's network config"
-  oc_obj:
-    state: list
-    kind: clusternetworks
-    name: default
-  register: clusternetworks
-  delegate_to: "{{ cluster_master }}"
-  run_once: true
-
 - set_fact:
-    kubernetes_node_name: "{{ (hostvars[inventory_hostname].ec2_private_dns_name if hostvars[inventory_hostname].oo_location == 'aws' else inventory_hostname) }}"
-    servicenetwork: "{{ clusternetworks.results.results[0].serviceNetwork }}"
+    kubernetes_node_name: "{{ ec2_private_dns_name if oo_location == 'aws' else inventory_hostname }}"
```

Comment:
Yes, we should never reference global variables (aka direct hostvars) in our roles.

Instead, our roles should have role variables and we should pass in the value to the role.

Here's an example:
https://github.com/openshift/openshift-ansible-ops/blob/prod/ops_tech_profiles/openshift_master/3.5/meta/main.yml#L18-L19